src/executor/aggregate/state.rs

```diff
@@ -90,13 +89,20 @@ impl<'a> State<'a> {
 
         values
             .into_iter()
-            .map(|(k, (_, v))| (k, v))
+            .map(|(k, v)| (k, v))
             .chunks(size)
             .into_iter()
             .enumerate()
             .map(|(i, entries)| {
                 let aggregated = entries
-                    .map(|((_, aggr), value)| (aggr, value))
+                    .map(|((_, aggr), v)| {
+                        //v.2 : exists to store the intermediate value of the avg function.
```

Comment:
Format comment 

```suggestion
                        // v.2 : exists to store the intermediate value of the avg function.
```
ref: <https://doc.rust-lang.org/reference/comments.html>

Reply:
Resolved commit: <506f34fc51e77e060450984253e7244c76d4cf8f>